virttest/qemu_devices/qdevices.py

```diff
@@ -1410,6 +1410,73 @@ def verify_unplug(self, out, monitor):  # pylint: disable=W0613,R0201
         return "\'%s\'" % self.get_qid() not in str(out)
 
 
+class QCPUDevice(QDevice):
+    """
+    CPU Device object, supports hot plug/unplug vcpu device with specified
+    properties.
+    """
+
+    def __init__(self, cpu_type, params=None, aobject=None, parent_bus=None):
```

Comment:
Miss doc string.

Reply:
`params=None, aobject=None, parent_bus=None` can reference the parent object, I added cpu_type document now.